ImperatorToCK3/Mappers/Localizaton/LocalizationMapper.cs

```diff
@@ -0,0 +1,169 @@
+                    continue;
+                }
+                var value = newLine.Substring(quoteLoc + 1, quote2Loc - quoteLoc - 1);
+
+                var gotValue = localizations.TryGetValue(key, out var locBlock);
+                if (gotValue) {
+                    if (language == "english")
+                        locBlock.english = value;
+                    if (language == "french")
+                        locBlock.french = value;
+                    if (language == "german")
+                        locBlock.german = value;
+                    if (language == "russian")
+                        locBlock.russian = value;
+                    if (language == "simp_chinese")
+                        locBlock.simp_chinese = value;
+                    if (language == "spanish")
+                        locBlock.spanish = value;
+                } else {
+                    var newBlock = new LocBlock();
+                    if (language == "english")
+                        newBlock.english = value;
+                    if (language == "french")
+                        newBlock.french = value;
+                    if (language == "german")
+                        newBlock.german = value;
+                    if (language == "russian")
+                        newBlock.russian = value;
+                    if (language == "simp_chinese")
+                        newBlock.simp_chinese = value;
+                    if (language == "spanish")
+                        newBlock.spanish = value;
+                    localizations.Add(key, newBlock);
+                }
+            }
+        }
+        public LocBlock? GetLocBlockForKey(string key) {
+            var gotValue = localizations.TryGetValue(key, out var locBlock);
+            if (!gotValue) {
+                return null;
+            }
+
+            if (!string.IsNullOrEmpty(locBlock.english) &&
+                !string.IsNullOrEmpty(locBlock.french) &&
+                !string.IsNullOrEmpty(locBlock.german) &&
+                !string.IsNullOrEmpty(locBlock.russian) &&
+                !string.IsNullOrEmpty(locBlock.simp_chinese) &&
+                !string.IsNullOrEmpty(locBlock.spanish)) {
+
+                var newBlock = locBlock;
+                if (string.IsNullOrEmpty(newBlock.french)) {
+                    newBlock.french = newBlock.english;
+                }
+                if (string.IsNullOrEmpty(newBlock.german)) {
+                    newBlock.german = newBlock.english;
+                }
+                if (string.IsNullOrEmpty(newBlock.russian)) {
+                    newBlock.russian = newBlock.english;
+                }
+                if (string.IsNullOrEmpty(newBlock.simp_chinese)) {
+                    newBlock.simp_chinese = newBlock.english;
+                }
+                if (string.IsNullOrEmpty(newBlock.spanish)) {
+                    newBlock.spanish = newBlock.english;
+                }
```

Comment:
These are all very similar, is it possible to use a single function you call for each language?

Reply:
Done, see LocBlock.FillMissingLocsWithEnglish and LocBlock.FillMissingLocWithEnglish.